uefi/flashregionsection.go

```diff
@@ -10,29 +10,44 @@ import (
 // FlashRegionSectionSize is the size of the Region descriptor. It is made up by 18 fields, each 16-bits large.
 const FlashRegionSectionSize = 36
 
+// Region contains a the start and end of a region in flash. This can be a BIOS, ME, PDR or GBE region.
+type Region struct {
```

Comment:
uint16? really? If it's a block number or something you might want to indicate in the comment, OR make a type to use. uint16 is confusing to me.

Reply:
done